core/src/main/kotlin/org/jetbrains/research/testspark/core/progress/CustomProgressIndicator.kt

```diff
@@ -9,6 +9,7 @@ interface CustomProgressIndicator {
     fun getFraction(): Double
     fun cancel()
     fun isCanceled(): Boolean
+    fun isStopped(): Boolean
```

Comment:
What is the difference between `isCanceled` and `isStopped`? As far as I know, IntelliJ's indicators only have a cancel button.

UPD: they seem to be different ([here](https://github.com/JetBrains/intellij-community/blob/master/platform/core-api/src/com/intellij/openapi/progress/ProgressIndicator.java)).